### DISCUSSION TITLE
📝【LeetCode】77 组合（n 个元素中选择 k 个）

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#76] [Announcements]
https://imageslr.github.io/2020/01/30/leetcode-77.html 

 问题描述 这道题是 LeetCode 77 题。

**Comment by Odinaris** [comment]
谢谢您的这篇博客，对我很有帮助。
不过有个小小的错误我想纠正一下，您在“附：01 转换法原理”部分中给的递推公式举例写错了，其中C(4,3)应该等于C(3,3)+C(3,2)而不是C(3,3)+C(4,2)。

**Comment by imageslr** [comment]
@Odinaris 谢谢指正🙏
